app/src/main/java/de/westnordost/streetcomplete/controls/QuestDownloadProgressFragment.kt

```diff
@@ -98,12 +100,13 @@ class QuestDownloadProgressFragment : Fragment(R.layout.fragment_quest_download_
             .start()
     }
 
+    @SuppressLint("UseCompatLoadingForDrawables")
     private fun updateDownloadProgress() {
         if (downloadProgressSource.isDownloadInProgress) {
             showProgressView()
             val questType = downloadProgressSource.currentDownloadingQuestType
             if (questType != null) {
-                progressView.setIcon(resources.getDrawable(questType.icon))
```

Comment:
What's wrong with `resources`?

Reply:
`Resources.getDrawable()` has been deprecated in favor of `Context.getDrawable()`.

Reply:
That's really weird, isn't the Resources class the thing to access any resources? Are drawables not resources anymore?

Reply:
My bad, it's deprecated in favor of `Resources.getDrawable(int, Theme)`.

Reply:
And why not use `Resources.getDrawable(in, Theme)` then?